snippets/contract-format.js

```diff
@@ -0,0 +1,58 @@
+// WARNING: THE CODE IN THIS FILE WILL BE SHOWN ON THE AGORIC
+// DOCUMENTATION SITE.
```

Comment:
thanks for adding a contract template. 

I don't think the warning comment is needed,  and it does show up in the expanded "details".

Reply:
Ah, that's a mistake. I need to use the region in the snippet import. Fixed. 

I like the warning, but if it might accidentally show up on the Agoric Documentation site, that's almost worse. Hmm. You think I should take it out even with the fix?

Reply:
I don't think it's necessary at all. Why do you find it helpful? 

Since all our code (at least in the public repositories :wink:) is publicly visible, I'm not sure what the extra exposure is that's worth a warning.

Reply:
Ok, sure, I'll take it out